src/Chapter1.hs

```diff
@@ -620,8 +633,10 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = div last2 10 + mod last2 10
```

Comment:
Great solution! You can note that `mod last2 10` id actually `lastDigit last2` function from the previous task! 😉